python/src/scipp/compat/mantid.py

```diff
@@ -82,25 +82,71 @@ def make_detector_info(ws):
                                         labels={'spectrum': spectrum}))
 
 
+def md_dimension(mantid_dim, index):
+    # Look for q dimensions
+    patterns = ["^q.*{0}$".format(coord) for coord in ['x', 'y', 'z']]
+    q_dims = [sc.Dim.Qx, sc.Dim.Qy, sc.Dim.Qz]
+    pattern_result = zip(patterns, q_dims)
+    if mantid_dim.getMDFrame().isQ():
+        for pattern, result in pattern_result:
+            if re.search(pattern, mantid_dim.name, re.IGNORECASE):
+                return result
+
+    # Look for common/known mantid dimensions
+    patterns = ["DeltaE", "T"]
+    dims = [sc.Dim.EnergyTransfer, sc.Dim.Temperature]
+    pattern_result = zip(patterns, dims)
+    for pattern, result in pattern_result:
+        if re.search(pattern, mantid_dim.name, re.IGNORECASE):
+            return result
+
+    # Look for common spacial dimensions
+    patterns = ["^{0}$".format(coord) for coord in ['x', 'y', 'z']]
+    dims = [sc.Dim.X, sc.Dim.Y, sc.Dim.Z]
+    pattern_result = zip(patterns, dims)
+    for pattern, result in pattern_result:
+        if re.search(pattern, mantid_dim.name, re.IGNORECASE):
+            return result
+
+    raise ValueError(
+        "Cannot infer scipp dimension from input mantid dimension {}".format(
+            mantid_dim.name()))
+
+
+def md_unit(frame):
+    known_md_units = {
+        "Angstrom^-1": sc.units.dimensionless / sc.units.angstrom,
+        "r.l.u": sc.units.dimensionless,
+        "T": sc.units.deg,
```

Comment:
```suggestion
        "T": sc.units.K,
```
?
`sc.unit.deg` is the angle unit, not a temperature unit.

Reply:
:) Of course